nautobot/dcim/tests/test_models.py

```diff
@@ -1338,20 +1164,6 @@ def test_tagged_vlan_raise_error_if_mode_not_set_to_tagged(self):
             err.exception.message_dict["tagged_vlans"][0], "Mode must be set to tagged when specifying tagged_vlans"
         )
 
-    def test_error_raised_when_adding_tagged_vlan_with_different_site_from_interface_parent_site(self):
```

Comment:
Should this become an equivalent test for Location mismatch instead of being removed?